src/environmentd/tests/pgwire.rs

```diff
@@ -470,7 +474,9 @@ fn test_arrays() {
     let message = client
         .simple_query("SELECT ARRAY[ARRAY[1], ARRAY[NULL::int], ARRAY[2]]")
         .unwrap()
-        .into_first();
+        .into_iter()
+        .find(|m| matches!(m, SimpleQueryMessage::Row(_)))
```

Comment:
Simple queries were changed and an additional message specifying the resulting schema is emitted first, so we must filter for the `Row` event instead of assuming it's the first one